src/main/scala/ru/dsr/bigdata/Tools.scala

```diff
@@ -1,26 +1,54 @@
 package ru.dsr.bigdata
 
+import org.apache.spark.rdd.RDD
 import org.apache.spark.sql._
+import ru.dsr.bigdata.Constants._
 
 object Tools {
 
-  private val options = Map("host" -> "localhost:27017", "database" -> "population")
+  def loadRddFromUrl(url: String, withHeader: Boolean)(implicit spark: SparkSession): RDD[Row] = {
 
-  def loadFromCsv(fileName: String)(implicit spark: SparkSession): DataFrame = {
+    val source = scala.io.Source.fromURL(url).mkString
+    var list = source.split("\n").filter(_ != "")
+
+    if (withHeader)
+      list = list.drop(1)
+
+    spark
+      .sparkContext
+      .parallelize(list
+        .map(r => r
+          .replace("\"", "")
+          .split(",")))
+      .map(
```

Comment:
How it will behave if we get `json` as `csv` field or `,` inside field value? It possibly removes mandatory quotes and splits line incorrectly, I guess.
Please, try to find more general and reliable solution for loading from remote `csv`. 
Also, signature for both `load` methods should be the same.